cross-modal-search/evaluate.py

```diff
@@ -88,11 +91,14 @@ def evaluation_generator(num_docs=None, batch_size=8, dataset_type='f8k'):
 
 
 def print_evaluation_score(resp):
-    sum_of_score = 0
+    batch_of_score = 0
     for doc in resp.search.docs:
-        sum_of_score += doc.evaluations[0].value
-        print(f'Evaluation {doc.evaluations[0].op_name}: {doc.evaluations[0].value}')
-    print(f'MeanReciprocalRank is: {sum_of_score/len(resp.search.docs)}')
+        batch_of_score += doc.evaluations[0].value
+    print(f'MeanReciprocalRank of the batch is: {batch_of_score/len(resp.search.docs)}')
+    global sum_of_score
+    global num_of_matches
+    sum_of_score += batch_of_score
+    num_of_matches += len(resp.search.docs)
```

Comment:
as a sanity check, can u sum the num of matches naively from the number of Evaluations u compute and see if it matches